build-farm/make-adopt-build-farm.sh

```diff
@@ -132,6 +132,11 @@ then
     # OpenJ9 only supports building jdk-11 with jdk-11
     JDK_BOOT_VERSION="11"
   fi
+  if [ "${JAVA_FEATURE_VERSION}" == "19" ]; then
+    # To support reproducible-builds the jar/jmod --date option is required
+    # which is only available from jdk-19
```

Comment:
Worth adding that clarification into the code I think:

```suggestion
    # which is only available from jdk-19 so we cannot bootstrap with JDK18
```